meta-resin-common/recipes-support/resin-init/resin-init-flasher.bb

```diff
@@ -68,3 +68,11 @@ do_install() {
     echo "BOOTLOADER_BLOCK_SIZE_OFFSET=${BOOTLOADER_BLOCK_SIZE_OFFSET}" >> ${D}/${sysconfdir}/resin-init-flasher.conf
     echo "BOOTLOADER_SKIP_OUTPUT_BLOCKS=${BOOTLOADER_SKIP_OUTPUT_BLOCKS}" >> ${D}/${sysconfdir}/resin-init-flasher.conf
 }
+
+do_deploy() {
+    if [ -z "EXTERNAL_DEVICE_UBOOT" ]; then
+        bbfatal "EXTERNAL_DEVICE_UBOOT is required in order to hardcode the initial uEnv.txt"
+    fi
+
+    echo -e "mmcdev=${EXTERNAL_DEVICE_UBOOT} \n bootpart=${EXTERNAL_DEVICE_UBOOT}:2" > ${DEPLOYDIR}/uEnv.txt
```

Comment:
Please remove the extra spaces in " \n ". I think this would break resinhup when checking different values as it uses foo.startswith()